webui/src/styles.css

```
@@ -389,6 +389,170 @@ a:focus-visible {
+}
+
+.card {
+  margin-bottom: 0.9rem;
+  padding: 0.9rem;
+  border: 1px solid color-mix(in srgb, CanvasText 22%, Canvas 78%);
+  border-inline-start: 0.25rem solid color-mix(in srgb, CanvasText 32%, Canvas 68%);
+  border-radius: 0.35rem;
+  background: color-mix(in srgb, Canvas 94%, CanvasText 6%);
+}
+
+.card--shell { border-inline-start-color: SlateBlue; }
```

Stylelint (17.14.0)

[error] Expected "SlateBlue" to be "slateblue" (value-keyword-case)

(value-keyword-case)